### PR TITLE
Adding Spark integration to UI

### PR DESCRIPTION
**Pull request by hsubbaraj-spiral**
## Describe your changes and why you are making these changes
Add Spark integration to UI
## Loom demo (if any)
Screen Shot 2023-02-23 at 2 54 02 PM

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/utils/integrations.ts [hunk 1]
```diff
@@ -429,6 +436,11 @@ export const SupportedIntegrations: ServiceInfoMap = {
     category: IntegrationCategories.NOTIFICATION,
     docs: `${addingIntegrationLink}/connecting-to-slack`,
   },
+  ['Spark']: {
+    logo: ServiceLogos['Spark'],
+    activated: true,
```
Comment (hsubbaraj-spiral) [inline src/ui/common/src/utils/integrations.ts:441]: Will change this to 'false' until full functionality is added!

File: src/ui/common/src/components/integrations/cards/sparkCard.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+   const config = integration.config as SparkConfig;
+   return (
+     <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+       <Typography variant="body2">
+         <strong>Livy Server URL: </strong>
+         {config.livy_server_url}
+       </Typography>
+     </Box>
+   );
+ };
```
Comment (agiron123) [inline src/ui/common/src/components/integrations/cards/sparkCard.tsx:23]: nit: at the bottom

```
export default SparkCard;
```

File: src/ui/common/src/utils/integrations.ts (hunk 1 above)
Comment (agiron123) [reply src/ui/common/src/utils/integrations.ts:441]: 👍

**Review by agiron123 — APPROVED** [review]
Nice work! Left some small comments here, but code looks good!

If you post some screenshots/loom I can leave comments on design stuff. (it's probably going to look the same as our other cards, but worth looking at).

**Pull request merged**